src/main/java/thaumicboots/api/ItemBoots.java

```diff
@@ -292,17 +292,18 @@ public static void renderHUDSpeedNotification() {
         Minecraft mc = Minecraft.getMinecraft();
         String text = getModeText(
                 "thaumicboots.speedEffect",
-                getBoots(mc.thePlayer).stackTagCompound.getDouble(TAG_MODE_SPEED) * 100);
+                IBoots.getBoots(mc.thePlayer).stackTagCompound.getDouble(TAG_MODE_SPEED) * 100);
         GTNHLib.proxy.printMessageAboveHotbar(text, 60, true, true);
     }
 
     @Optional.Method(modid = "gtnhlib")
     @SideOnly(Side.CLIENT)
     public static void renderHUDOmniNotification() {
         Minecraft mc = Minecraft.getMinecraft();
-        String result = "thaumicboots.omniState" + getBoots(mc.thePlayer).stackTagCompound.getBoolean(TAG_MOD_OMNI);
+        String result = "thaumicboots.omniState"
+                + IBoots.getBoots(mc.thePlayer).stackTagCompound.getBoolean(TAG_MOD_OMNI);
```

Comment:
and here

Reply:
getBoots() returns null with non ItemBoots item (EMT boots)

This will cause a clientside NPE crash when changing omni.